standalone/ansible/roles/redis/tasks/main.yml

```diff
@@ -1,22 +0,0 @@
----
-- name: Redis | Change default incoming policy changed to 'deny'
-  ufw:
-    direction: incoming
-    default: deny
-
-- name: Redis | Allow SSH
-  ufw:
-    rule: allow
-    name: OpenSSH
-
-- name: Redis | Allow access to redis to servers (webservers + sidekiq) only
-  ufw:
-    rule: allow
-    proto: tcp
-    src: "{{ item }}"
-    to_port: "{{ redis_port }}"
-  with_items: "{{ groups.servers }}"
-
-- name: Redis | Enable ufw
-  ufw:
-    state: enabled
```

Comment:
Is this removed because it's included for all boxes in `main` now?

Reply:
Yes, added in the `common` role.

Reply:
Oops, yes I meant `common` 👍